ezra-project.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
```

Comment:
Should this be the actual Ezra Project version?

Reply:
No, this is the version of the desktop entry specification file format. It affects what fields are required, how categories are parsed, etc.

Reply:
Spec details here: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html#basic-format